lib/Handler/CompletionHandler.php

```diff
@@ -96,10 +96,21 @@ public function completion(TextDocumentItem $textDocument, Position $position):
 
     public function registerCapabiltiies(ServerCapabilities $capabilities)
     {
-        $capabilities->completionProvider = new CompletionOptions(false, [':', '>']);
+        $capabilities->completionProvider = new CompletionOptions(false, [':', '>', '$']);
         $capabilities->signatureHelpProvider = new SignatureHelpOptions(['(', ',']);
     }
 
+    private function formatLabel(Suggestion $suggestion): string
+    {
+        switch ($suggestion->type()) {
+            case Suggestion::TYPE_VARIABLE:
+                return mb_substr($suggestion->label(), 1);
+            default:
+                return $suggestion->label();
```

Comment:
I would remove `default` and move the return to be after the switch statement (so the switch is a guard I guess)